src/ansys/tools/path/path.py

```diff
@@ -9,11 +9,14 @@
 
 import platformdirs
 
+from ansys.tools.path.applications import ApplicationPlugin, dyna, mapdl, mechanical
 from ansys.tools.path.misc import is_float, is_linux, is_windows
 
+PLUGINS: Dict[str, ApplicationPlugin] = {"mechanical": mechanical, "dyna": dyna, "mapdl": mapdl}
```

Comment:
I wonder how we are going to populate this with new applications. You did mention. the example for 7zip, and I guess "7zip" should appear here. Somehow.

I guess we need to use something to store the new apps?? Maybe a txt file in a settings directory? Something like this?

https://github.com/ansys/pymapdl/blob/1a720959ab3e91dce20074b514d4faeb7707e11a/src/ansys/mapdl/core/__init__.py#L32-L37

Reply:
@germa89 this is just an implementation detail. Ideally there would be a way to install plugins. I would want to learn about the `pluggy` library in python which seems very well suited for adding plugins.

In the case of 7zip or other "unregistered" applications, those would not have plugins. The whole point here is to allow `save` & `get` for anything, but `find` only for registered applications.

Reply:
Ok I see. Fine with me then.